modules/congestion_controller/alpha_cc/alpha_cc_network_control.cc

```diff
@@ -94,7 +96,37 @@ NetworkControlUpdate GoogCcNetworkController::OnSentPacket(
 
 NetworkControlUpdate GoogCcNetworkController::OnStreamsConfig(
     StreamsConfig msg) {
-  return GetDefaultState(msg.at_time);
+  NetworkControlUpdate update;
+  update.target_rate = TargetTransferRate();
+  update.target_rate->network_estimate.at_time = msg.at_time;
+  update.target_rate->network_estimate.bandwidth = last_received_bandwidth_estimate_;
+  update.target_rate->network_estimate.loss_rate_ratio =
+      last_estimated_fraction_loss_ / 255.0;
+  update.target_rate->network_estimate.round_trip_time = TimeDelta::Millis(last_estimated_rtt_ms_);
+
+  TimeDelta default_bwe_period = TimeDelta::Seconds(3);  // the default is 3sec
+  update.target_rate->network_estimate.bwe_period = default_bwe_period;
+  update.target_rate->at_time = msg.at_time;
+  update.target_rate->target_rate = last_received_bandwidth_estimate_;
+
+  //*-----Set pacing & padding_rate-----*//
+  int32_t default_pacing_rate = last_received_pacing_rate_;   // default:300000;=> 750000 bps = 750 kbps
+  int32_t default_padding_rate = 0;  // default: 0bps = 0kbps
+  DataRate pacing_rate = DataRate::BitsPerSec(default_pacing_rate * pacing_factor_);
+  DataRate padding_rate = DataRate::BitsPerSec(default_padding_rate);
+  PacerConfig message;
+  message.at_time = msg.at_time;
+  message.time_window = TimeDelta::Seconds(1);
+  message.data_window = pacing_rate * message.time_window;
+  message.pad_window = padding_rate * message.time_window;
+
+  update.pacer_config = message;
+
+  // *-----Set congestion_window-----*//
+  update.congestion_window = current_data_window_;
+
+  return update;
+  // return GetDefaultState(msg.at_time);
```

Comment:
same as previous one, delete the unnecessary code if you don't need them anymore instead of commenting them